DEVELOPMENT.md

```diff
@@ -353,12 +358,43 @@ be 403 Forbidden. In some cases, the status code can also be 404 Not Found (e.g.
 if a user is trying to access a resource that exists but to which he does not
 have access).
 
+#### Scopes
+
+In addition to the role of the user, the JWTs issued when authenticating have a
+list of allowed scopes. Each protected API endpoint **must** have a
+corresponding scope, e.g. `api:users:create` for `POST /api/users`. If a
+protected API call is made with a JWT that does not have the correct scope, the
+response's status code will be 403 Forbidden. In some cases, the status code can
+also be 404 Not Found (e.g.  if a user is trying to access a resource that
+exists but to which he does not have access).
+
+Scopes are hierarchical. If a JWT contains a top-level scope, it is allowed to
+access all children scopes. For example, if a JWT has the `api:users` scope, it
+is authorized to access API endpoints with the `api:foo:create` or the
+`api:foo:update` scope, but not with the `api:bar:create` scope.
+
+> This allows JWTs to be generated with limited access to specific API
+> resources or operations.
+
+Default JWTs issued when a user logs in have the `api` scope, authorizing
+everything the user can normally do through the API. Special additional scopes
+may be created to allow specific operations.
+
+This is the list of authorization scopes in the API:
+
+Scope                   | Description
+:---                    | :---
+`api:locations:create`  | Authorizes access to `POST /api/locations`.
+`api:locations:update`  | Authorizes access to `PATCH /api/locations`.
+`api:locations:destroy` | Authorizes access to `DELETE /api/locations`.
```

Comment:
Not useful now, although it might be in the future (e.g. if we add an administration backend). I basically added a scope for each write operations.

However, I agree to remove them for the sake of [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it). I will do that in the next PR since I will update this section anyway when I add user updates.